src/internal/FloatingMenu.js

```diff
@@ -318,21 +318,26 @@ class FloatingMenu extends React.Component {
   _getChildrenWithProps = () => {
     const { styles, children } = this.props;
     const { floatingPosition: pos } = this.state;
-    return !pos
-      ? children
-      : React.cloneElement(children, {
-          style: Object.assign(
-            {
-              left: `${pos.left}px`,
-              top: `${pos.top}px`,
-              position: 'absolute',
-              right: 'auto',
-              margin: 0,
-              opacity: 1,
-            },
-            styles
-          ),
-        });
+    const style = pos
```

Comment:
As a code style comment, could we re-write this in a way that could communicate why these values are needed? Comments are fine as well 🙂

As an added note, why in this ternary do we apply `styles` when `pos` is defined but not when it isn't defined? Trying to catch up on this component implementation 😄

Reply:
@joshblack updated. Should be a little clearer now.

I was thinking I didn't want to apply stylings if we're just hiding it off to the side but you're right, if those stylings change the size of the element, that should be included. Updated.